README.md

```diff
@@ -29,10 +29,18 @@ ip 0.0.0.0
 rcon_password mac_rcon
 net_start 
 ```
-> Note: the rcon_password is subject to change. If you use loopback (127.0.0.1) for the rcon IP, you are prevented from joining community servers.
+
+### Notes:
+
+> The rcon_password is subject to change. If you use loopback (127.0.0.1) for the rcon IP, you are prevented from joining community servers.
+
+> Be aware that if either the 'net_start' command or your 'autoexec' config with 'net_start' are executed multiple times during a single game, this can result in the game's networking being reset.
+
+> The rcon command manager relies on accessing the port 27015, which causes issues if another application is using it. iTunes (AppleDeviceServices) is a notable application that binds to this port. 
+
 
 You will also need to provide a SteamAPI key to the client. The client looks for configs in a config folder specified by OS.
-- Windows: `C:\Users\<Your Name>\AppData\Roaming\Mega Anti Cheat\config`
+- Windows: `C:\Users\<Your Name>\AppData\Roaming\MAC\MACClient\config`
 - Mac OS: TODO - work out what this is 
 - Linux: `~/.config/megaanticheatclient/`
```

Comment:
The linux directory is now `~/.config/macclient/` if you wouldn't mind updating that as well. Happy to merge after that.